llvm/lib/CodeGen/SelectionDAG/LegalizeIntegerTypes.cpp

```diff
@@ -1804,6 +1816,7 @@ bool DAGTypeLegalizer::PromoteIntegerOperand(SDNode *N, unsigned OpNo) {
   case ISD::FP16_TO_FP:
   case ISD::VP_UINT_TO_FP:
   case ISD::UINT_TO_FP:   Res = PromoteIntOp_UINT_TO_FP(N); break;
+  case ISD::STRICT_FP16_TO_FP:
```

Comment:
This just doesn't look goo imo. Why not add PromoteIntOp_STRICT_FP16_TO_FP? Or rename PromoteIntOp_STRICT_UINT_TO_FP?

Reply:
The non-strict path does the same thing, so I'm just copying that. I don't see much point in renaming it